e2e-tests/cypress/e2e/pages/your-business/companies-house-search/companyDetails.js

```diff
@@ -0,0 +1,28 @@
+const companyDetails = {
+  errorSummary: () => cy.get('[data-cy="error-summary'),
+  headingCaption: () => cy.get('[data-cy="heading-caption'),
```

Comment:
Can we put this in partials - the heading caption will be in every single page moving forwards so